lib/close_old_pull_requests.rb

```diff
@@ -24,11 +22,11 @@ def initialize(pull_requests)
     end
 
     def outdated
-      pull_requests.group_by { |pr| [pr[:title], pr[:user][:login]] }.values.map do |pulls|
+      pull_requests.group_by { |pr| [pr[:title], pr[:user][:login]] }.values.flat_map do |pulls|
         pulls = pulls.sort_by { |p| p[:created_at] }.reverse
-        new_pr = PullRequest.new(pulls.first[:number])
-        pulls.drop(1).map { |p| PullRequest.new(p[:number], new_pr) }
-      end.compact.flatten
+        new_pr = pulls.first
+        pulls.drop(1).map { |pull| [pull, new_pr] }
```

Comment:
I think this would be a lot clearer with a `pulls.shift` rather than a `pulls.first` and `pulls.drop(1)`